docs/groovy-script/mods/selectionguicrafting/sgc_recipe.md

```diff
@@ -0,0 +1,249 @@
+---
+title: "Crafting Recipe"
+titleTemplate: "Selection GUI Crafting | CleanroomMC"
+description: "Creates a recipe that is shown in the specified category. It can requires an input, output and tool. You may also need a catalyst to craft the recipe."
+source_code_link: "https://github.com/Ender-Development/selection-gui-crafting-continued/blob/master/src/main/java/io/enderdev/selectionguicrafting/integration/groovyscript/SgcRecipe.java"
+---
+
+# Crafting Recipe (Selection GUI Crafting)
+
+## Description
+
+Creates a recipe that is shown in the specified category. It can requires an input, output and tool. You may also need a catalyst to craft the recipe.
+
+## Identifier
+
+Refer to this via any of the following:
+
+```groovy:no-line-numbers {1}
+mods.selectionguicrafting.sgc_recipe/* Used as page default */ // [!code focus]
+mods.selectionguicrafting.sgcrecipe
+mods.selectionguicrafting.sgcRecipe
+mods.selectionguicrafting.SgcRecipe
+```
+
+
+## Adding Recipes
+
+### Recipe Builder
+
+Just like other recipe types, the Crafting Recipe also uses a recipe builder.
+
+Don't know what a builder is? Check [the builder info page](../../getting_started/builder.md) out.
+
+:::::::::: details mods.selectionguicrafting.sgc_recipe.newRecipe() {open id="abstract"}
+- `Integer`. The amount of XP that will be granted to the player when the recipe is crafted. Requires greater than or equal to 0. (Default `0`).
+
+    ```groovy:no-line-numbers
+    xp(int)
+    ```
+
+- `Integer`. The time in ticks it takes to craft the recipe. Requires greater than or equal to 0. (Default `0`).
+
+    ```groovy:no-line-numbers
+    time(int)
+    ```
+
+- `ArrayList<GsTool>`. The tool that will be used to craft the recipe. Tool refers to the mainhand item. This doesn't need to be an item with durability. Requires greater than or equal to 1.
+
+    ```groovy:no-line-numbers
+    tool(IIngredient)
+    tool(IIngredient, float)
+    tool(IIngredient, float, float)
+    ```
+
+- `ResourceLocation`. The path to the frame texture. Overrides the category frame.
+
+    ```groovy:no-line-numbers
+    frame(String)
+    ```
+
+- `ArrayList<Ingredient>`. The input items that are required to craft the recipe. Input refers to the offhand item. Requires greater than or equal to 1.
```

Comment:
`Input refers to the offhand item` that sounds like it should be included in the main description.